.github/workflows/tests.yml

```diff
@@ -48,3 +48,36 @@ jobs:
         with:
           name: doc-html-${{ matrix.branch }}
           path: www/
+
+  translations:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest", "windows-latest"]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3
+          allow-prereleases: true
+          cache: pip
+      - name: Install dependencies
+        run: |
+          pip install -U pip setuptools
+          pip install Babel jinja2
+      - run: python3 babel_runner.py extract
+      - run: python3 babel_runner.py init -l pt_BR
+      - run: python3 babel_runner.py update
+      - run: python3 babel_runner.py update -l pt_BR
+      - run: python3 babel_runner.py compile
+      - run: python3 babel_runner.py compile -l pt_BR
```

Comment:
Here you are using python3 and using 

> os: ["ubuntu-latest", "windows-latest"] 

to run on multiple systems. But python3 doesn't work on my Windows, use python?

Reply:
That won't be necessary, as [actions/setup-python](https://github.com/actions/setup-python) installs python3 binary (or symlink to binary, I don't know), so those commands work fine in the CI environment. See [this Windows workflow run in my fork](https://github.com/rffontenelle/python-docs-theme/actions/runs/10221587570/job/28284437047).

Any documentation we might write will probably need to take your scenario in consideration.

Reply:
Great🚀

Reply:
We already use `python` rather than `python3` in _this_ repo, so I suggest we keep it the same here.

Reply:
Ok, change applied. It looks like there are 3 votes on `python` (AA-Turner too) :D